library/common/Chart.yaml

```
@@ -15,7 +15,7 @@ maintainers:
 name: common
 sources: null
 type: library
-version: 17.5.1
+version: 18.0.0
```

```suggestion
version: 17.5.1
```

Remove pump so we can merge test changes